sensor/Instructions.md

```diff
@@ -51,14 +51,25 @@ make updaterdk
 
 Next, run `make bin/customsensor` to build the executable.
 
-#### 6. Test your module locally
+#### 5. Test your module locally
 
-Follow the instructions to [Add a local module](https://docs.viam.com/registry/configure/#add-a-local-module).
+While testing, we recommend that you add the resources you've built with a remote to your machine because it allows you to recompile your code without having to restart your entire machine:
 
-#### 7. Upload your module to the registry
+1. Edit the [`cmd/module/cmd.go` file](./cmd/module/cmd.go). The code there should configure and run a robot with your modular resource configured.
+2. If you haven't already got a machine, create a new machine on [the Viam app](app.viam.com) and follow the setup instructions on the setup tab.
+3. Run `make remote` to compile the [`cmd/module/cmd.go` file](./cmd/module/cmd.go).
```

Comment:
`make remote` doesn't compile the module... `make bin/customsensor` does.

Reply:
this should all say `cmd/remote/cmd.go`